doc/nrf/ug_thread_ot_integration.rst

```diff
@@ -0,0 +1,130 @@
+.. _openthread_integration:
+
+OpenThread Integration
+######################
+
+.. contents::
+    :local:
+    :depth: 2
+
+The goal of this page is to explain how the OpenThread stack is integrated with remaining parts of the system in terms of the file structure,
+used threads and synchronization primitives and the network traffic flow.
+
+The OpenThread stack is integrated with the Zephyr RTOS as the L2 layer with the IEEE 802.15.4 Radio Driver underneath and Zephyr IPv6 stack above it.
+The main advantage of this approach is that it can utilize the Zephyr's L3 layer which allows for easy easy application porting and utilization of UDP, TCP and ICMP build on top of Zephyr's IPv6 stack.
+The drawback, on the other hand, lies in the reception path in which the IP packet needs to traverse the OpenThread L3 layer and then Zephyr's L3 layer in order to reach the BSD socket.
+
+.. note:: If one desires the application can use just the OpenThread API with it's IPv6 stack and even the internal CoAP implementation which is the application protocol.
```

Comment:
```suggestion
.. note:: The application can use the OpenThread API and it's IPv6 stack directly, including the internal CoAP implementation which is the application protocol.
```

Reply:
This part could be more precise.
Using OpenThread's IPv6 is not that easy. However using the UDPv6 based on that is.

I will just commit this version with `and it's UDPv6/IPv6 stack directly`.
If you think it should be rephrased please let me know.